app/models/like.rb

```diff
@@ -0,0 +1,13 @@
+class Like < ApplicationRecord
+  has_many :likes, foreign_key: 'posts_id'
+  has_many :comments, foreign_key: 'posts_id'
+  belongs_to :author, class_name: 'User', dependent: :destroy, foreign_key: 'author_id'
+
+  def update_posts_couter
```

Comment:
Great job @eerapheal 🤝

- [ ] I would suggest you change this method `update_posts_counter` to `update_likes_counter` since we need a method to update the likes counter for a post.

Reply:
Thanks for the review. @1GORDON